src/routes/account/account.controller.ts

```diff
@@ -0,0 +1,64 @@
+import { Request, Response } from 'express';
+import { Account } from '../../schemas/account';
+
+export async function getAccount(
+  req: Request,
+  res: Response
+) {
+  try{
+    const docs =  await Account.findById(req.query.userId)
+    if(!docs) {
+      res.status(404);
+      return res.json();
+    }
+    res.status(200);
+    return res.json(docs);
+  } catch (e) {
+    res.status(400);
+    return res.send();
+  }
+  
+}
+
+export async function saveAccount(
+  req: Request,
+  res: Response
+) {
+
+  console.log(req.body);
+  try {
+    const savedAccount = await Account.create(req.body);
+    if(savedAccount) {
+    const userId = savedAccount._id;
+    res.status(201);
+    return res.json({ userId });
+    } else {
+      throw new Error();
+    }
+  } catch (e) {
+    res.status(401);
```

Comment:
```suggestion
    res.status(400);
```
HTTP 401 ist Unauthorized, du würdest hier daher einen 400 (Client Error) oder 500 (Internal Server Error) werfen, wahrscheinlich eher 500, weil du nicht sicher sein kannst, dass der Fehler wirklich beim Anfrager liegt, glaube aber in eurer Schnittstellendoku steht 400, aber ziemlich sicher kein 401